Windows.Devices.WiFi/Windows.Devices.Wifi.sln

```diff
@@ -1,17 +1,9 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27130.2027
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.31129.286
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{11A8DD76-328B-46DF-9F39-F559912D0360}") = "Windows.Devices.Wifi", "Windows.Devices.Wifi\Windows.Devices.Wifi.nfproj", "{A42BC265-9F96-48E2-AA2A-A5119B9B0B79}"
-EndProject
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{19A726FD-5E93-45A7-AD59-A79EFE734567}"
-	ProjectSection(SolutionItems) = preProject
```

Comment:
Please don't remove the solution folder and the items contained there.

Reply:
Sorry.
Removing the .sln file has been suggested in my past efforts.  Thought it might fix the a200000 error.

Reply:
No problem. That's what code reviews are for! 😉

Reply:
Do I need to do anything or can we 'resolve' & try the build again?

Reply:
This won't build as it is... suggest that you revert all changes **except** the one on the WiFiAdapter class

Reply:
Will do

Reply:
Just pushed the original .sln file in it's original location